src/components/ButtonAdd.js

```diff
@@ -0,0 +1,36 @@
+import React, { useState, useEffect } from 'react'
+import { BrowserRouter as Router, Link } from 'react-router-dom'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import styled from 'styled-components'
+import { NavLink } from 'react-router-dom'
+
+export default function ButtonAdd() {
+  const [toggle, setToggle] = useState(true)
```

Comment:
Maybee it is better to place the hook into the app.js?
I am not sure what is best practice.